packages/node_modules/@webex/test-users/src/generate-random-string.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Generates a random string
+ * @name generateRandomString
+ * @param  {Integer} length
+ * @returns {String} A random string of specified length
+ */
+export default function generateRandomString(length) {
+  const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
+  let text = '';
+
+  for (let i = 0; i < length; i += 1) {
+    text += chars.charAt(Math.floor(Math.random() * chars.length));
+  }
+
+  return text;
+}
```

Comment:
Would it be better to use the UUID utility here?

Reply:
I'll give it a shot, the only place it is used is for generating a password.